bot/cogs/error_handler.py

```diff
@@ -236,6 +236,8 @@ def __init__(self, bot: Bot):
             f"```{e.__class__.__name__}: {e}```"
         )
 
+        ctx.bot.stats.incr("command_error_count")
```

Comment:
Let's name this `commands.error_count` to be consistent with `commands.{command_name}`.

Reply:
I've avoided that intentionally. Since we may want something like "Total executed commands" and for that we can look at the metric `commands.*` (wildcard) and get all command invocations.

Reply:
That's fair. How about `errors.commands`, in case we add more error stats in the future?

Reply:
:okK_hand: